_posts/blog/2018-11-12/2018-11-12-how-will-the-bch-hard-fork-on-nov-15-affect-openbazaar.markdown

```diff
@@ -0,0 +1,33 @@
+---
+title: "How Will The <span class='post-title-extra'>BCH Hard Fork</span> Affect OpenBazaar?"
+layout: post
+date: '2018-11-12 06:30:00 -0300'
+social_title: 'How Will The Bitcoin Cash (BCH) Hard Fork on November 15th Affect OpenBazaar?'
+social_description: 'Here is what you need to know about the Bitcoin Cash hard fork on November 15th as it relates to Openbazaar.'
+---
+
+#### What’s Happening with Bitcoin Cash?
+
+There is a [Bitcoin Cash](https://www.bitcoincash.org/) network upgrade due to complete at 4:40PM UTC on November 15th, 2018. Depending on how the upgrade is accepted by the network, the Bitcoin Cash blockchain could fork into two or more incompatible chains. In order to help OpenBazaar users prepare for the fork, here is some information about the OpenBazaar wallet that can help you decide how to handle any BCH funds that you may have stored there.
+
+#### How Will The Bitcoin Cash Upgrade Affect My OpenBazaar Funds?
```

Comment:
You should mention that transacting in BCH during a fork is dangerous unless it's between addresses your entirely control. If you transact BCH to a an address you don't control you could lose access to your coins on other forks.

Reply:
Updated: There is a [Bitcoin Cash](https://www.bitcoincash.org/) network upgrade beginning at 4:40PM UTC on November 15th, 2018. Depending on how the upgrade is accepted by the network, the Bitcoin Cash blockchain could fork into two or more incompatible chains. 

It's important to note that transacting is dangerous during a hard fork like this unless it's between addresses you entirely control. If you transact BCH to an address you don't control you could lose access to your coins on other forks. 

In order to help OpenBazaar users prepare for the fork, here is some information about the OpenBazaar wallet that can help you decide how to handle any BCH funds that you may have stored there.